client/signup/steps/store-features/index.tsx

```diff
@@ -145,6 +147,9 @@ export default function StoreFeaturesStep( props: Props ): React.ReactNode {
 			defaultDependencies={ {
 				siteTitle: '',
 				tagline: '',
+				//Temporarily hard-coded to Zoologist; eventually we'll add a theme selection step.
+				themeSlugWithRepo: 'pub/zoologist',
```

Comment:
I'm not certain this is actually working; it looks like Zoologist may be the default theme used when any site signs up without selecting a theme.

Reply:
Good thing to verify – but once you know that's the case, no harm leaving it there in case something gets changed elsewhere.

Reply:
Yeah, we're already using Zoologist by default at signup if someone manages to skip the theme selection step: https://github.com/Automattic/wp-calypso/blob/44cd4083641d58827ad3acdc7a1b5071c549f9c4/client/lib/signup/site-type.js#L7

Reply:
Actually, let's not leave any unnecessary code lying around; if by default users are getting Zoologist, let's leave it at that.